app/controllers/media_controller.rb

```diff
@@ -76,6 +74,22 @@ def requested_from_draft_assets_host?
     request.host == AssetManager.govuk.draft_assets_host
   end
 
+  def draft_asset_manager_access?
+    token = params.fetch(:token, cookies[:auth_bypass_token])
+    payload = decode_auth_bypass_token(token)
+
+    return false if payload == false
+
+    payload["draft_asset_manager_access"].present? && payload["draft_asset_manager_access"] == true
```

Comment:
As you're checking for true here you could simplify this whole line down to `payload["draft_asset_manager_access"] == true`